phoenix-core/src/it/java/org/apache/phoenix/rpc/UpdateCacheIT.java

```diff
@@ -244,4 +248,25 @@ private static void validateSelectRowKeyCols(Connection conn, String selectSql,
         }
         assertFalse(rs.next());
     }
+
+    @Test
+    public void testInvalidConnUpdateCacheFrequencyShouldThrow() throws Exception {
+        Properties props = PropertiesUtil.deepCopy(TestUtil.TEST_PROPERTIES);
+
+        ArrayList<String> invalidUCF = new ArrayList<>();
+        invalidUCF.add("GIBBERISH");
+        invalidUCF.add("10000.6");
+
+        for (String connLevelUCF : invalidUCF) {
+            props.put(QueryServices.DEFAULT_UPDATE_CACHE_FREQUENCY_ATRRIB, connLevelUCF);
+            try {
+                DriverManager.getConnection(getUrl(), props);
+                fail();
+            } catch (IllegalArgumentException e) {
+                // expected
+                assertEquals("Connection's phoenix.default.update.cache.frequency can only be set to 'ALWAYS', 'NEVER' or a millisecond numeric value.",
```

Comment:
Same with comparing the message string here.

Reply:
Okay, same here.